### PR TITLE
update readme & makefile

### PR DESCRIPTION
**Pull request by elgnay**
Signed-off-by: Yang Le <yangle@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,25 +15,135 @@ You can reach the maintainers of this project at:
 -->
 
 ------
+## Description
+
+### Work agent
+Work agent is a controller running on the managed cluster. It watches the `ManifestWorks` CRs in the cluster namespace on hub cluster and applies the manifests included in those CRs on the managed clusters.
```
Comment (qiujian16) [inline README.md]: we need to define what is cluster namespace. Work agent does not require a cluster namespace notion here actually. You can say that work agent will monitor the work resource on a certain ns on hub, in which the work agent has permission.

File: README.md [hunk 2]
```diff
@@ -15,25 +15,135 @@ You can reach the maintainers of this project at:
...
+kubectl --kubeconfig </path/to/hub_cluster/kubeconfig> -n cluster1 get manifestwork/hello-work -o yaml
+```
+
+Check on the managed cluster and see the `Pod` has been deployed from the hub cluster.
+```
+kubectl --kubeconfig </path/to/managed_cluster/kubeconfig> -n default get pod
+NAME                     READY   STATUS    RESTARTS   AGE
+hello-64dd6fd586-rfrkf   1/1     Running   0          108s
+```
+
```
Comment (qiujian16) [inline README.md:190]: I would add also some explanation on the status field in the work here. For instance, what is manifestConditions and what is workCondition. What does it mean by applied/availbale.

Comment (elgnay) [reply README.md:190]: Explanation on status added.

File: README.md (hunk 1 above)
Comment (elgnay) [reply README.md]: Doc updated.

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**